csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -7,11 +7,13 @@
  *   csc f1.cs f2.cs f3.cs
  *
  * The `id` simply identifies the invocation, while `cwd` is the working
- * directory from which the compiler was invoked.
+ * directory from which the compiler was invoked. The `assembly` identifies
+ * the output assembly of the compilation.
  */
 compilations(
     unique int id : @compilation,
-    string cwd : string ref
+    string cwd : string ref,
+    int assembly: @assembly ref
```

Comment:
The `compilations` relation is shared between multiple languages (currently C++, C# and Java). In the interest of maintaining compatibility between languages, have you considered implementing the additional information as a separate relation? In Java, for example, a compilation typically results in multiple `.class` files, not a single assembly.

Reply:
I have no preference. I'll change it to a separate relation. How is the `compilations` shared between languages?

Reply:
Manual sync between dbscheme files at this time. :see_no_evil: